### PR TITLE
Add eslint and prettier

### PR DESCRIPTION
**Pull request by VolhaVaitulevich**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true,
+		node: true,
+	},
+	extends: [
+		'eslint:recommended',
+		'plugin:react/recommended',
```
Comment (siarhei-papkou) [inline .eslintrc.js]: Do we need react here?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		'prettier',
+	],
+	overrides: [],
+	parser: '@typescript-eslint/parser',
+	parserOptions: {
+		ecmaVersion: 'latest',
+		sourceType: 'module',
+	},
+	plugins: ['react', '@typescript-eslint'],
+	rules: {
```
Comment (siarhei-papkou) [inline .eslintrc.js:19]: let's add `@typescript-eslint/explicit-function-return-type` rule

**Review by siarhei-papkou — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (VolhaVaitulevich) [reply .eslintrc.js:19]: Added this in rules with 'off' and in overrides to enable this rule for .ts and .tsx files only (if this is redundant and it should be just in rules, please let me know)

File: .eslintrc.js (hunk 1 above)
Comment (VolhaVaitulevich) [reply .eslintrc.js]: No, it's not required, it's not a React project, I removed it in a new commit.

**Review by siarhei-papkou — APPROVED** [review]

**Pull request merged**